mtools/mplotqueries/plottypes/scatter_type.py

```diff
@@ -6,6 +6,8 @@
 from mtools.mplotqueries.plottypes.base_type import BasePlotType
 
 try:
+    import matplotlib
```

Comment:
Is this import necessary given the plottypes are loaded by way of `mplotqueries.py`?